extensions/objects/models/objects_test.py

```diff
@@ -35,13 +35,12 @@ def check_normalization(
         """Test that values are normalized correctly.
 
         Args:
-            object_class: child of BaseObject. The class whose normalize()
+            object_class: object(BaseObject). The class whose normalize()
                 method is to be tested.
-            mappings: a list of 2-element tuples. The first element of
+            mappings: list(tuple(str, str)). The first element of
                 each item is expected to be normalized to the second.
-            invalid_items_with_error_messages: a list of values and
-                corresponding error messages. Each of the value is expected to
-                raise an Exception when normalized.
+            invalid_items_with_error_messages: list(tuple(str, str)). Each of
```

Comment:
You removed the first part of docstring: a list of values and corresponding error messages